synapse/appservice/api.py

```diff
@@ -80,10 +78,6 @@
 
 HOUR_IN_MS = 60 * 60 * 1000
 
-
-APP_SERVICE_PREFIX = "/_matrix/app/v1"
```

Comment:
Keeping the prefix I think is still useful / more consistent with what we do in other places.

Reply:
For future reference, besides consistency are there any heuristics I should keep in mind about when to use a constant vs when to hardcode something like this case here?

Reply:
Usually if you're putting the same text several times it is worth having a constant.